masterlist.yaml

```diff
@@ -34013,3 +34020,21 @@ plugins:
         util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 4
         udr: 40
+
+  - name: 'DLCFrostcragCoblAlchemyTable.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/52608/'
+        name: 'Frostcrag Spire Cobl Alchemy Station'
+    req: [ *Cobl ]
+    after: [ 'DLCFrostcrag - Unofficial Patch.esp' ]
+    tag: [ Scripts ]
+    dirty:
+      # Version: 0.9
+      - <<: *quickClean
+        crc: 0x802681D8
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        udr: 4
+    clean:
+      # Version 1.0
```

Comment:
```suggestion
```

Version comments aren't necessary, the version number can be noted in the commit message.